.ci/kitchen-centos7-py2-cloud

```diff
@@ -1,14 +1,14 @@
-@Library('salt@master-1.5') _
+@Library('salt@master-1.6') _
 
 runTestSuite(
-    concurrent_builds: 0,
+    concurrent_builds: 0,  // Don't cancel builds still running. Wait until they end.
```

Comment:
That means that if a new cloud build rolls in it won't cancel the existing one, right? i.e. if I create a PR it will launch one of these tests, and then I make a change, it will have two tests?

Or are these only run on branch builds?

Reply:
Yes, we're not canceling previous runs on purpose, and yes, cloud tests only run on branch builds.